app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -171,14 +164,7 @@ class PatientSummaryScreen(context: Context, attrs: AttributeSet) : RelativeLayo
 
   private fun setupEditButtonClicks() {
     editButton.setOnClickListener {
-      val (key, architecture) = if (mobiusMigrationConfig.useEditPatientScreen) {
-        createEditPatientScreenKey(patientSummaryProfile!!) to Architecture.MOBIUS
-      } else {
-        createPatientEditScreenKey(patientSummaryProfile!!) to Architecture.ORIGINAL
-      }
-      Analytics.reportArchitectureMigration("EditPatientScreen", architecture, null)
```

Comment:
We can remove `Analytics#reportArchitectureMigration` too now since this was the last usage left. 🕺

Reply:
Done.